core/formats/formats_10.cpp

```diff
@@ -2842,7 +2843,7 @@ doc_id_t wanderator<IteratorTraits, FieldTraits, WandIndex, WandExtent,
       std::get<position<IteratorTraits, FieldTraits>>(attrs_).notify(notify);
     }
 
-    target = doc.value + 1;
+    seek_target = doc_value + 1;
```

Comment:
We will only decrease the current target in the "tail" case, but in this case it doesn't matter.

Reply:
It's probably fix seek in the tail for this iterator

Reply:
I don't know how, in general it's in the same case when while needed for next in the doc_iterator